tools/admin/wskadmin

```diff
@@ -576,7 +576,7 @@ def getLimitsCmd(args, props):
             print('No limits found, default system limits apply')
         else:
             print('Failed to get limits (%s)' % res.read().strip())
-        return 1
+            return 1
```

Comment:
can you cover all three exits? return 0 at line 576, and leave return 1 on 579?

Reply:
I did the change, anything else still missing?